UncorEncounterModel.m

```diff
@@ -0,0 +1,535 @@
+classdef UncorEncounterModel < EncounterModel
```

Comment:
Similar comments as `EncounterModel`. If this is a copy, use the existing version. If this was modified, please streamline